README.md

```diff
@@ -63,6 +63,37 @@ If no `binding` is provided then tracing is ignored and the logger falls back to
 
 > Note: you can hide the binding necessity behind an abstraction layer if you prefer
 
+### Logging Levels
+
+The logger includes an option to define a desired logging level. Only log messages that are higher than the desired level will be logged.
```

Comment:
Equal to or higher than surely?

Reply:
Sorry, I don't follow. Do you mean I should add "equal to or higher"?